packages/round-manager/src/features/round/ViewRoundPage.tsx

```diff
@@ -88,40 +86,40 @@ export default function ViewRound() {
                     <Tab
                       className={({ selected }) =>
                         selected ?
-                        'border-grey-500 text-grey-500 whitespace-nowrap py-4 px-1 border-b-2 font-bold text-sm outline-none' :
-                        'border-transparent text-grey-400 hover:text-gray-700 hover:border-gray-300 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm'
+                        "border-grey-500 text-grey-500 whitespace-nowrap py-4 px-1 border-b-2 font-bold text-sm outline-none" :
+                        "border-transparent text-grey-400 hover:text-gray-700 hover:border-gray-300 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm"
                       }
                     >
                       Received
                     </Tab>
                     <Tab
                       className={({ selected }) =>
                         selected ?
-                        'border-grey-500 text-grey-500 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm outline-none' :
-                        'border-transparent text-grey-400 hover:text-gray-700 hover:border-gray-300 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm'
+                        "border-grey-500 text-grey-500 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm outline-none" :
+                        "border-transparent text-grey-400 hover:text-gray-700 hover:border-gray-300 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm"
                       }
                     >
                       Approved
                     </Tab>
                     <Tab
                       className={({ selected }) =>
                         selected ?
-                        'border-grey-500 text-grey-500 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm outline-none' :
-                        'border-transparent text-grey-400 hover:text-gray-700 hover:border-gray-300 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm'
+                        "border-grey-500 text-grey-500 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm outline-none" :
+                        "border-transparent text-grey-400 hover:text-gray-700 hover:border-gray-300 whitespace-nowrap py-4 px-1 border-b-2 font-medium text-sm"
                       }
                     >
                       Rejected
                     </Tab>
                   </Tab.List>
                   <Tab.Panels>
                     <Tab.Panel>
-                      <ApplicationsReceived />
+                      <ListApplications status="PENDING" />
                     </Tab.Panel>
                     <Tab.Panel>
-                      <ApplicationsApproved />
+                    <ListApplications status="APPROVED" />
                     </Tab.Panel>
                     <Tab.Panel>
-                      <ApplicationsRejected />
+                    <ListApplications status="REJECTED" />
```

Comment:
Hmm this might work for now but we do have bulk approval coming in next / possible appeal flow 
At that point , having everything within the same   `ListApplications` might make the file too heavy ?
Would it be easier to keep em separate from the start ? (aka how it originally was as 3 independent components)

Reply:
done ✅